sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -700,6 +693,60 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  // TODO should we change this config?
+  val ENABLE_CSV_TIMESTAMPS = conf("spark.rapids.sql.csvTimestamps.enabled")
+      .doc("When set to true, enables the CSV parser to read timestamps. The default output " +
+          "format for Spark includes a timezone at the end. Anything except the UTC timezone is " +
+          "not supported. Timestamps after 2038 and before 1902 are also not supported.")
+      .booleanConf
+      .createWithDefault(false)
+
+  val ENABLE_READ_CSV_DATES = conf("spark.rapids.sql.csv.read.date.enabled")
+      .doc("Parsing invalid CSV dates produces different results from spark")
+      .booleanConf
+      .createWithDefault(false)
+
+  val ENABLE_READ_CSV_BOOLS = conf("spark.rapids.sql.csv.read.bool.enabled")
+      .doc("Parsing invalid CSV boolean values produce true instead of null")
```

Comment:
```suggestion
      .doc("Parsing an invalid CSV boolean value produces true instead of null")
```

Reply:
This comment was missed in the update

Reply:
that pesky `s` should have been added now.